### PR TITLE
Utne new branch

### PR DESCRIPTION
**Pull request by Vonutne**
Finished selcetScreen, and did some minor clean up

### CODE REVIEW

**Comment by Vonutne** [comment]
Selectscreen

File: src/scivu/scivu/ViewModels/VersionViewModel.cs
Comment (NicolajRJensen) [inline src/scivu/scivu/ViewModels/VersionViewModel.cs:1]: Is this file used? There's no accompanying view

File: src/scivu/scivu/Views/SelectSurveyMenuView.axaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:vm="using:scivu.ViewModels"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="vm:SelectSurveyMenuViewModel"
+             x:Class="scivu.Views.SelectSurveyMenuView">
+  <DockPanel>
+    <StackPanel DockPanel.Dock="Top" HorizontalAlignment="Left" >
```
Comment (NicolajRJensen) [inline src/scivu/scivu/Views/SelectSurveyMenuView.axaml:10]: Should probably be Center aligned like the rest of the views.
```suggestion
    <StackPanel DockPanel.Dock="Top" HorizontalAlignment="Center" >

```

File: src/scivu/scivu/ViewModels/VersionViewModel.cs
Comment (Vonutne) [reply src/scivu/scivu/ViewModels/VersionViewModel.cs:1]: It is used for the SurveySelectViewModel, like the SurveyViewModel so yes

File: src/scivu/scivu/Views/SelectSurveyMenuView.axaml (hunk 1 above)
Comment (Vonutne) [reply src/scivu/scivu/Views/SelectSurveyMenuView.axaml:10]: Was intentional to have it to the side but can make it centered :)

**Review by NicolajRJensen — APPROVED** [review]
LGTM

**Pull request merged**